tests/collections/asr/test_label_datasets.py

```diff
@@ -80,3 +82,78 @@ def test_tarred_dataset_duplicate_name(self, test_data_dir):
         for _ in ds_list_load:
             count += 1
         assert count == 6
+
+    @pytest.mark.unit
+    def test_feat_seqlabel_dataset(self, test_data_dir):
+        manifest_path = os.path.abspath(os.path.join(test_data_dir, 'asr/feat/emb.json'))
+        feature_loader = ExternalFeatureLoader(
+            file_path=manifest_path, sample_rate=16000, int_values=False, augmentor=None
+        )
+        ds_braceexpand = FeatureToSeqSpeakerLabelDataset(
+            manifest_filepath=manifest_path, labels=self.unique_labels_in_seq, feature_loader=feature_loader
+        )
+
+        correct_label = torch.tensor(
+            [
```

Comment:
Can you disable formatting here and just flatten this list ? Use `#fmt: off` and `#fmt: on` before and after respective to this tensor creation.